formatting.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: Formatting Flutter code
+sidebar: home_sidebar
+permalink: /formatting/
+---
+
+Flutter source code style guidelines, and support for automatic formatting.
+
+* TOC Placeholder
+{:toc}
+
+## Flutter formatting guidelines
```

Comment:
I think we can remove this from this article. It really belongs in the CONTRIBUTING.md or similar document. This isn't useful for most people, only really for contributors to the SDK itself.